src/Lecture1.hs

```diff
@@ -123,4 +142,15 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> String
+lowerAndGreater n list = go n (0 :: Integer) (0 :: Integer) list
+    where go n' lower greater [] = show n' ++ " is greater than " ++ show greater ++ " elements and lower than " ++ show lower ++ " elements"
```

Comment:
You can use standard function `unwords` to join a list of strings with spaces. This way it can look a bit nicer:

```suggestion
  where
    go n' lower greater [] = unwords
        [ show n'
        , "is greater than"
        , show greater
        , "elements and lower than"
        , show lower
        , "elements"
        ]
```